### PR TITLE
ref(starfish): require referrer in discover hooks

### PR DESCRIPTION
**Pull request by DominikB2014**
Every discover query should have a referrer

### CODE REVIEW

File: static/app/views/aiMonitoring/PipelinesTable.tsx [hunk 1]
```diff
@@ -100,39 +100,44 @@ export function PipelinesTable() {
...
+      search: new MutableSearch(
+        `span.category:ai span.ai.pipeline.group:[${(data as Row[])?.map(x => x['span.group']).join(',')}]`
+      ),
+      fields: [
+        'span.ai.pipeline.group',
+        'ai_total_tokens_used()',
+        'ai_total_tokens_used(c:spans/ai.total_cost@usd)',
+      ],
+    },
+    'use-span-metrics-default-please-update'
```
Comment (gggritso) [inline static/app/views/aiMonitoring/PipelinesTable.tsx]: 😂 you can choose something sensible here like `api.performance.ai-analytics.token-usage-chart` or whatever

File: static/app/views/aiMonitoring/PipelinesTable.tsx [hunk 2]
```diff
@@ -128,7 +130,9 @@ export function PipelinesTable() {
       'ai_total_tokens_used()',
       'ai_total_tokens_used(c:spans/ai.total_cost@usd)',
     ],
-  });
+  },
+    'use-span-metrics-default-please-update'  ); // TODO - update this reff
```
Comment (DominikB2014) [inline static/app/views/aiMonitoring/PipelinesTable.tsx:134]: @colin-sentry could you provide me a ref to use for this?

File: static/app/views/performance/queues/messageConsumerSamplesPanel.tsx [hunk 3]
```diff
@@ -75,11 +75,14 @@ export function MessageConsumerSamplesPanel() {
...
-    yAxis: [`avg(span.self_time)`],
-    enabled: isPanelOpen,
-  });
+  } = useSpanMetricsSeries(
+    {
+      search,
+      yAxis: [`avg(span.self_time)`],
+      enabled: isPanelOpen,
+    },
+    'use-span-metrics-series-default-please-update' // TODO - update this referrer
```
Comment (DominikB2014) [inline static/app/views/performance/queues/messageConsumerSamplesPanel.tsx:84]: @edwardgou-sentry could you suggest a ref to use here?

Comment (gggritso) [inline static/app/views/performance/queues/messageConsumerSamplesPanel.tsx:84]: Same here, any reasonable default is fine

**Review by gggritso — APPROVED** [review]
Makes sense, just one note about adding some default referrers

File: static/app/views/aiMonitoring/PipelinesTable.tsx [hunk 4]
```diff
@@ -100,35 +100,40 @@ export function PipelinesTable() {
...
+      search: new MutableSearch(
+        `span.category:ai span.ai.pipeline.group:[${(data as Row[])?.map(x => x['span.group']).join(',')}]`
+      ),
+      fields: [
+        'span.ai.pipeline.group',
+        'ai_total_tokens_used()',
+        'ai_total_tokens_used(c:spans/ai.total_cost@usd)',
+      ],
+    },
+    'use-span-metrics-default-please-update' // TODO - update this reff
```
Comment (DominikB2014) [inline static/app/views/aiMonitoring/PipelinesTable.tsx:135]:
```suggestion
    'api.performance.ai-analytics.token-usage-chart' // TODO - update this reff
```

File: static/app/views/performance/queues/messageConsumerSamplesPanel.tsx (hunk 3 above)
Comment (DominikB2014) [inline static/app/views/performance/queues/messageConsumerSamplesPanel.tsx:84]:
```suggestion
    'api.performance.queues.avg-duration-chart'
```

**Pull request merged**